subsys/bluetooth/host/l2cap_internal.h

```diff
@@ -214,14 +214,8 @@ struct net_buf *bt_l2cap_create_pdu_timeout(struct net_buf_pool *pool,
  *
  * Buffer ownership is transferred to stack in case of success.
  */
-int bt_l2cap_send_cb(struct bt_conn *conn, uint16_t cid, struct net_buf *buf,
-		     bt_conn_tx_cb_t cb, void *user_data);
-
-static inline int bt_l2cap_send(struct bt_conn *conn, uint16_t cid,
-				struct net_buf *buf)
-{
-	return bt_l2cap_send_cb(conn, cid, buf, NULL, NULL);
-}
+int bt_l2cap_send_pdu(struct bt_l2cap_le_chan *le_chan, struct net_buf *seg,
```

Comment:
Any reason you're calling it `seg` rather than `pdu`? The latter would match the function name.

Reply:
remnants of when it was called `send_seg()`. Will rename.